packages/client/src/Pages/Forum/components/TopicsList/TopicsList.module.scss

```
@@ -0,0 +1,32 @@
+.Topics {
+  grid-area: main;
+}
+
+.TopicsList {
+  font-size: var(--font-size-30);
+  margin: 70px 0;
+}
+
+.TopicItem {
+  & + & {
+    margin-top: 55px;
+  }
+
+  &:hover span,
+  &:focus span {
+    color: var(--sky);
+  }
+}
+
+.Topic {
+  display: grid;
+  grid-template-columns: calc(2 / 3 * 1fr) calc(1 / 3 * 1fr);
```

можно сделать проще:
```suggestion
  grid-template-columns: 2fr 1fr;
```